### PR TITLE
[Docs] Fix typos, improve, update at Using Diffusers' Loading & Hub page

### PR DESCRIPTION
**Pull request by standardAI**
# What does this PR do?

Hi! Thank you for this awesome library!
I am new to open source. Firstly, I am reading the documentation. Secondly, I will read the codebase. Thirdly, I plan to make documentation and code contributions. I am open to your guidance.
I would be grateful for your feedback. @stevhliu @yiyixuxu @patrickvonplaten

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [X] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/en/using-diffusers/other-formats.md [hunk 1]
```diff
@@ -144,6 +151,8 @@ pipeline = DiffusionPipeline.from_pretrained(
 ).to("cuda")
 pipeline.scheduler = UniPCMultistepScheduler.from_config(pipeline.scheduler.config)
 ```
+<!--Repository Not Found for url: https://huggingface.co/api/models/andite/anything-v4.0.
+Please make sure you specified the correct `repo_id` and `repo_type`.-->
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/other-formats.md:155]: Looks like the model was removed. I think we can update the code snippet to use `runwayml/stable-diffusion-v1-5`, and then remove this comment. cc @sayakpaul do you know of any other cool model we can replace this with?

File: docs/source/en/using-diffusers/push_to_hub.md [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
 # Push files to the Hub
 
 [[open-in-colab]]
+<!--Notebook links don't work.
+-->
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/push_to_hub.md:17]:
```suggestion
```

File: docs/source/en/using-diffusers/push_to_hub.md [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
 
 🤗 Diffusers provides a [`~diffusers.utils.PushToHubMixin`] for uploading your model, scheduler, or pipeline to the Hub. It is an easy way to store your files on the Hub, and also allows you to share your work with others. Under the hood, the [`~diffusers.utils.PushToHubMixin`]:
 
-1. creates a repository on the Hub
-2. saves your model, scheduler, or pipeline files so they can be reloaded later
-3. uploads folder containing these files to the Hub
+1. creates a repository on the Hub;
+2. saves your model, scheduler, or pipeline files so they can be reloaded later;
+3. uploads folder containing these files to the Hub.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/push_to_hub.md:23]: No need for the semicolons at the end since it is a list

File: docs/source/en/using-diffusers/schedulers.md [hunk 4]
```diff
@@ -241,7 +258,7 @@ image
 </p>
 
 
-At the time of writing this doc [`DPMSolverMultistepScheduler`] gives arguably the best speed/quality trade-off and can be run with as little 
+At the time of November 15th, 2022 [`DPMSolverMultistepScheduler`] gives arguably the best speed/quality trade-off and can be run with as little
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/schedulers.md:261]: Let's go with this to avoid dating the doc. cc @patrickvonplaten @sayakpaul @yiyixuxu is this statement still valid?

```suggestion
[`DPMSolverMultistepScheduler`] gives one of the best speed/quality trade-offs and can be run with as little 
```

**Review by stevhliu — COMMENTED** [review]
Thanks for taking the time to review the docs and run the code!

File: docs/source/en/using-diffusers/other-formats.md (hunk 1 above)
Comment (sayakpaul) [reply docs/source/en/using-diffusers/other-formats.md:155]: @apolinario

File: docs/source/en/using-diffusers/schedulers.md (hunk 4 above)
Comment (sayakpaul) [reply docs/source/en/using-diffusers/schedulers.md:261]: I would write "a reasonable trade-off".

File: docs/source/en/using-diffusers/loading_adapters.md [hunk 5]
```diff
@@ -202,6 +213,8 @@ Then fuse this pipeline with the next set of LoRA weights:
 pipeline.load_lora_weights("ostris/super-cereal-sdxl-lora")
 pipeline.fuse_lora(lora_scale=0.7)
 ```
+<!--The current API is supported for operating with a single LoRA file. You are trying to load and fuse more than one LoRA which is not well-supported.
+-->
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply docs/source/en/using-diffusers/loading_adapters.md:217]: Yeah, that works for me. 

We want to promote the usage of our `peft` integration as it is much more feature-complete. https://huggingface.co/docs/diffusers/main/en/tutorials/using_peft_for_inference

File: docs/source/en/using-diffusers/loading.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ This guide will show you how to load:
 
 </Tip>
 
-The [`DiffusionPipeline`] class is the simplest and most generic way to load any diffusion model from the [Hub](https://huggingface.co/models?library=diffusers). The [`DiffusionPipeline.from_pretrained`] method automatically detects the correct pipeline class from the checkpoint, downloads and caches all the required configuration and weight files, and returns a pipeline instance ready for inference.
+The [`DiffusionPipeline`] class is the simplest and most generic way to load any diffusion model from the [Hub](https://huggingface.co/models?library=diffusers&sort=downloads). The [`DiffusionPipeline.from_pretrained`] method automatically detects the correct pipeline class from the checkpoint, downloads and caches all the required configuration and weight files, and returns a pipeline instance ready for inference.
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply docs/source/en/using-diffusers/loading.md:36]: That works for me. Or maybe even just a note saying "if you want to know what's latest and trending, you could also use this link ..."

[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/loading.md]: Changed to trending.

File: docs/source/en/using-diffusers/loading.md [hunk 7]
```diff
@@ -96,19 +96,31 @@ from diffusers import DiffusionPipeline
 repo_id = "runwayml/stable-diffusion-v1-5"
 stable_diffusion = DiffusionPipeline.from_pretrained(repo_id, use_safetensors=True)
 stable_diffusion.scheduler.compatibles
+[diffusers.schedulers.scheduling_euler_ancestral_discrete.EulerAncestralDiscreteScheduler,
```
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/loading.md:99]: It was like that on this page:
https://huggingface.co/docs/diffusers/main/en/stable_diffusion#:~:text=pipeline.scheduler.compatibles

File: docs/source/en/using-diffusers/loading_adapters.md [hunk 8]
```diff
@@ -13,14 +13,20 @@ specific language governing permissions and limitations under the License.
 # Load adapters
 
 [[open-in-colab]]
+<!--Notebook links don't work.
+-->
```
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/loading_adapters.md]: There is not a corresponding notebook in the Hugging Face's notebooks repo.

File: docs/source/en/using-diffusers/loading_adapters.md [hunk 9]
```diff
@@ -62,16 +69,17 @@ Now you can load the textual inversion embeddings with the [`~loaders.TextualInv
...
+Textual Inversion can also be trained on undesirable things to create *negative embeddings* to discourage a model from generating images with those undesirable things like blurry images or extra fingers on a hand. This can be a easy way to quickly improve your prompt. You'll also load the embeddings with [`~loaders.TextualInversionLoaderMixin.load_textual_inversion`], but this time, you'll need two more parameters:
 
-- `weight_name`: specifies the weight file to load if the file was saved in the 🤗 Diffusers format with a specific name or if the file is stored in the A1111 format
-- `token`: specifies the special word to use in the prompt to trigger the embeddings
+- `weight_name`: Specifies the weight file to load if the file was saved in the 🤗 Diffusers format with a specific name or if the file is stored in the A1111 format.
+- `token`: Specifies the special word to use in the prompt to trigger the embeddings.
```
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/loading_adapters.md]: Reversed.

File: docs/source/en/using-diffusers/loading_adapters.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/loading_adapters.md:211]: Ok, understood.

File: docs/source/en/using-diffusers/other-formats.md [hunk 10]
```diff
@@ -64,9 +64,9 @@ git checkout pr/13
 
 3. There are several input arguments to configure in the conversion script, but the most important ones are:
 
-    - `checkpoint_path`: the path to the `.ckpt` file to convert.
-    - `original_config_file`: a YAML file defining the configuration of the original architecture. If you can't find this file, try searching for the YAML file in the GitHub repository where you found the `.ckpt` file.
-    - `dump_path`: the path to the converted model.
+    - `checkpoint_path`: The path to the `.ckpt` file to convert.
```
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/other-formats.md]: Reversed.

File: docs/source/en/using-diffusers/other-formats.md [hunk 11]
```diff
@@ -94,14 +94,21 @@ git push origin pr/13:refs/pr/13
 optimization techniques](https://huggingface.co/docs/diffusers/optimization/fp16).
 
 The [Convert KerasCV](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) Space converts `.pb` or `.h5` files to PyTorch, and then wraps them in a [`StableDiffusionPipeline`] so it is ready for inference. The converted checkpoint is stored in a repository on the Hugging Face Hub.
+<!--Convert KerasCV:
+runtime error
+failed to create containerd task: failed to create shim task: context deadline exceeded: unknown
+
+Container logs:
+
+Failed to retrieve error logs: Failed to load logs: Not Found. Logs are persisted for 30 days after the Space stops running.-->
```
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/other-formats.md]: Removed.

File: docs/source/en/using-diffusers/loading.md [hunk 12]
```diff
@@ -29,11 +29,11 @@ This guide will show you how to load:
...
+💡 Skip to the [DiffusionPipeline explained](#diffusionpipeline-explained) section if you are interested in learning in more detail about how the [`DiffusionPipeline`] class works.
 
 </Tip>
 
-The [`DiffusionPipeline`] class is the simplest and most generic way to load any diffusion model from the [Hub](https://huggingface.co/models?library=diffusers). The [`DiffusionPipeline.from_pretrained`] method automatically detects the correct pipeline class from the checkpoint, downloads and caches all the required configuration and weight files, and returns a pipeline instance ready for inference.
+The [`DiffusionPipeline`] class is the simplest and most generic way to load any diffusion model from the Hub. If you want to know what's latest and trending, you could also use [this link](https://huggingface.co/models?library=diffusers&sort=trending). The [`DiffusionPipeline.from_pretrained`] method automatically detects the correct pipeline class from the checkpoint, downloads, and caches all the required configuration and weight files, and returns a pipeline instance ready for inference.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/loading.md:36]:
```suggestion
The [`DiffusionPipeline`] class is the simplest and most generic way to load the latest trending diffusion model from the [Hub](https://huggingface.co/models?library=diffusers&sort=trending). The [`DiffusionPipeline.from_pretrained`] method automatically detects the correct pipeline class from the checkpoint, downloads, and caches all the required configuration and weight files, and returns a pipeline instance ready for inference.
```

File: docs/source/en/using-diffusers/loading.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/en/using-diffusers/loading.md:99]: I think its more appropriate on that page as a Get Started, but here, for the Loading guide I don't think we need to include the outputs which can be distracting for users trying to just do one thing

File: docs/source/en/using-diffusers/loading.md [hunk 13]
```diff
@@ -121,6 +133,9 @@ from diffusers import DiffusionPipeline
 
 repo_id = "runwayml/stable-diffusion-v1-5"
 stable_diffusion = DiffusionPipeline.from_pretrained(repo_id, safety_checker=None, use_safetensors=True)
+"""
```
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/en/using-diffusers/loading.md:136]: Actually, this should be ok :)

File: docs/source/en/using-diffusers/loading_adapters.md [hunk 14]
```diff
@@ -184,7 +189,7 @@ pipeline = StableDiffusionXLPipeline.from_pretrained(
 ).to("cuda")
 ```
 
-Then load the LoRA checkpoint and fuse it with the original weights. The `lora_scale` parameter controls how much to scale the output by with the LoRA weights. It is important to make the `lora_scale` adjustments in the [`~loaders.LoraLoaderMixin.fuse_lora`] method because it won't work if you try to pass `scale` to the `cross_attention_kwargs` in the pipeline. 
+Then, load the LoRA checkpoint and fuse it with the original weights. The `lora_scale` parameter controls how much to scale the output by with the LoRA weights. It is important to make the `lora_scale` adjustments in the [`~loaders.LoraLoaderMixin.fuse_lora`] method because it won't work if you try to pass `scale` to the `cross_attention_kwargs` in the pipeline.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/loading_adapters.md:192]:
```suggestion
Next, load the LoRA checkpoint and fuse it with the original weights. The `lora_scale` parameter controls how much to scale the output by with the LoRA weights. It is important to make the `lora_scale` adjustments in the [`~loaders.LoraLoaderMixin.fuse_lora`] method because it won't work if you try to pass `scale` to the `cross_attention_kwargs` in the pipeline. 
```

File: docs/source/en/using-diffusers/push_to_hub.md (hunk 2 above)
Comment (stevhliu) [reply docs/source/en/using-diffusers/push_to_hub.md:17]: This should be removed, we're aware of this issue

File: docs/source/en/using-diffusers/loading.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (standardAI) [reply docs/source/en/using-diffusers/loading.md:99]: Ok, reversed.

**Comment by patrickvonplaten** [comment]
Feel free to merge once good for you @stevhliu

File: docs/source/en/using-diffusers/other-formats.md (hunk 1 above)
Comment (stevhliu) [reply docs/source/en/using-diffusers/other-formats.md:148]: @standardAI, lets update this section with the below. You can probably keep most of the text, and just update the code snippets. The updated image URL is https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/blueprint-lora.png.

```py
from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_pretrained(
    "Lykon/dreamshaper-xl-1-0", torch_dtype=torch.float16, variant="fp16"
).to("cuda")

# uncomment to download the safetensor weights
!wget https://civitai.com/api/download/models/168776 -O blueprintify.safetensors

pipeline.load_lora_weights("/content/blueprintify.safetensors", weight_name="blueprintify.safetensors")
prompt = "bl3uprint, a highly detailed blueprint of the empire state building, explaining how to build all parts, many txt, blueprint grid backdrop"
negative_prompt = "lowres, cropped, worst quality, low quality, normal quality, artifacts, signature, watermark, username, blurry, more than one bridge, bad architecture"

images = pipeline(
    prompt=prompt,
    negative_prompt=negative_prompt,
    generator=torch.manual_seed(0),
).images[0]
images
```

Comment (standardAI) [reply docs/source/en/using-diffusers/other-formats.md]: Done.

**Review by stevhliu — APPROVED** [review]
Very nice, thanks for all the fixes!
